BridgeAppSDK/SBAPermissionsManager.m

```diff
@@ -490,7 +490,20 @@ - (BOOL)isPermissionsGrantedForCoreMotion {
 #if TARGET_IPHONE_SIMULATOR
     return YES;
 #else
-    return self.coreMotionPermissionStatus == SBAPermissionStatusAuthorized;
+    BOOL permission;
+    
+    // available iOS 9 and later: a way to check motion sensor recorder permissions without explicitly requesting them.
+    // TODO: emm 2016-07-12 redo the whole permissions thing so this situation doesn't even come up. We should
+    // be requesting permissions either ahead of time or as we first need them, but not asking the user for
+    // permission to even *ask* them for the permissions, which winds up with not being able to set them in Settings
+    // later if they don't allow us to ask the first time through.
+    if ([CMSensorRecorder class]) {
```

Comment:
The class doesn't exist in iOS 8 but since we're linking against iOS SDK 9 it knows about it and this is the recommended way to check for the availability of weakly-linked classes in iOS:

http://stackoverflow.com/questions/3057325/weak-linking-check-if-a-class-exists-and-use-that-class/10068079#10068079
https://developer.apple.com/library/ios/documentation/DeveloperTools/Conceptual/cross_development/Using/using.html#//apple_ref/doc/uid/20002000-SW3